third-party/riscv-arch-tests/src/riscv-target/cartesi/model_test.h

```diff
@@ -1,19 +1,19 @@
-// Copyright Cartesi and individual authors (see AUTHORS)
-// SPDX-License-Identifier: LGPL-3.0-or-later
-//
-// This program is free software: you can redistribute it and/or modify it under
-// the terms of the GNU Lesser General Public License as published by the Free
-// Software Foundation, either version 3 of the License, or (at your option) any
-// later version.
-//
-// This program is distributed in the hope that it will be useful, but WITHOUT ANY
-// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-// PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
-//
-// You should have received a copy of the GNU Lesser General Public License along
-// with this program (see COPYING). If not, see <https://www.gnu.org/licenses/>.
-//
-
+// Copyright Cartesi and individual authors (see AUTHORS)

```

Comment:
What changed here? I don't see any change. Is this an 'end-of-line' thingy?

Reply:
removed trailing white spaces